.github/workflows/build.yml

```diff
@@ -78,26 +62,3 @@ jobs:
         with:
           command: check
           args: --workspace --all-targets --all-features
-  dependency:
```

Comment:
Why we are removing them?

Reply:
Same as `features`, we don't have any deepest for now.